test/data/vessel-valid/environment-with inside heatIndexTemperature.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I just noticed that the earlier test data reorganisation added lots of files with spaces in their names. They are sort of a pain to work with on the command line, adding cruft to how they look, and I would be much happier with the spaces replaced with underscores, here and elsewhere.

Reply:
Fair call. See PR #449 which replaces all the spaces with underscores